File: milestone 2/python_modules/module00/ex5/ft_count_harvest_recursive.cpp
#include <iostream>

void ft_count_harvest_recursive(int day, int i) 
{
	if (i >= day)
	{
		std::cout << "Day " << day << "\n";
		ft_count_harvest_recursive(day + 1, i);
	}
	else if (i <= day)
		std::cout << "Harvest time!\n";
}

int main(void)
{
	int input;
	int day;

	std::cout << "Days until harvest: ";
	std::cin >> input;

	ft_count_harvest_recursive(1, input);
    return 0;
}

File: milestone 2/python_modules/module00/ex5/ft_count_harvest_recursive.py
def ft_count_harvest_recursive(day, i):
    if i >= day:
        print(f"Day {day}")
        ft_count_harvest_recursive(day + 1, i)
    elif i <= day:
        print("Harvest time!")


if __name__ == "__main__":
    print("Days until harvest: ")
input_val = int(input())
ft_count_harvest_recursive(1, input_val)
